contracts/PushCore/PushCoreV2_5.sol

```diff
@@ -94,7 +94,6 @@ contract PushCoreV2_5 is Initializable, PushCoreStorageV1_5, PausableUpgradeable
         if (
             !(
                 (channels[_channel].channelState == 1 && msg.sender == _channel)
-                    || (msg.sender == pushChannelAdmin && _channel == address(0x0))
```

Comment:
This line is not removed for the temp contract.

Reply:
this is now resolved @Zartaj0 . Thanks